.github/workflows/docker.yaml

```diff
@@ -90,6 +92,18 @@ jobs:
         uses: docker/metadata-action@v5.5.1
         with:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          flavor: |
```

Comment:
I trust this to work :D

Reply:
So do I,  needed quite a lot of tuning to get this right.